document/src/vespa/document/util/bytebuffer.h

```diff
@@ -134,17 +100,13 @@ class ByteBuffer
 
     void incPosNoCheck(size_t pos) {
         _pos += pos;
-#ifdef __FORCE_VALGRIND_ON_SERIALIZE__
-        forceValgrindStart2Pos();
-#endif
     }
 
     /**
      * Resets pos to 0, and sets limit to old pos. Use this before reading
      * from a buffer you have written to
      */
     void flip() {
-        _limit = _pos;
         _pos = 0;
```

Comment:
Consider updating comment to reflect new semantics

Reply:
Code removed